hack/for-go-proj.sh

```diff
@@ -19,58 +19,39 @@ set -o pipefail
 set -o nounset
 
 CONTRIB_ROOT="$(dirname ${BASH_SOURCE})/.."
-PROJECT_NAMES=(addon-resizer vertical-pod-autoscaler)
 
 if [[ $# -ne 1 ]]; then
-  echo "missing subcommand: [build|install|test]"
+  echo "missing subcommand: [cluster-autoscaler|vertical-pod-autoscaler|addon-resizer]"
   exit 1
 fi
 
-CMD="${1}"
-
-case "${CMD}" in
-  "build")
+PROJECT="${1}"
+
+case "${PROJECT}" in
```

Comment:
Given that the logic is I wonder if we should go a step further and just have a per-project file instead of keeping a single entry point here? If not, could we at least rename this file to something more accurate (e.g. `run-unit-tests.sh`)?